apps/designer/app/designer/features/style-panel/style-source/style-source-input.tsx

```diff
@@ -191,20 +196,59 @@ type StyleSourceInputProps<Item> = {
   css?: CSS;
 };
 
+const newItemId = "__NEW__";
```

Comment:
is it impossible to add 2 new items and have a conflicting id? otherwise I would just generate one

Reply:
Nope, it's special item. Created in one place, used in other. Need to be in the list to make downshift work.